vector/src/main/java/im/vector/app/features/spaces/create/CreateSpaceViewModel.kt

```diff
@@ -126,10 +126,12 @@ class CreateSpaceViewModel @AssistedInject constructor(
             SpaceTopology.JustMe -> {
                 setState {
                     copy(
-                            spaceTopology = SpaceTopology.JustMe
+                            spaceTopology = SpaceTopology.JustMe,
+                            defaultRooms = emptyMap()
                     )
                 }
                 // XXX finish and open the add rooms directly
+                handleNextFromDefaultRooms()
```

Comment:
`XXX` can be removed fro comment (it's like a TODO, no?)

Reply:
fixed, thx